### ISSUE TITLE
Gpu hang when equipping a tool

### ISSUE THREAD

**Issue by Thehuggblepanda** [Yellow-Dog-Man/Resonite-Issues#2217]
### Describe the bug?

Whenever I use the protoflux tool to drive values like rotation and position a driver in my gpu crashes and causes the game to not render anything visually. I'm still in the game, people can hear me, I can hear people and move and interact with things.

### To Reproduce

No real way where things can be reproduced since it seems to be and issue specifically with my system it seems.
Nvidia 4060 8gb vram      Graphics drivers version: 555.85
32gb ddr4 ram
i5-10400F                          Windows 11

### Expected behavior

Equip the ProtoFlux tool, Grab the rotation values on any object, attempt to drive the values to use in the ProtoFlux coding system. Screen hangs for over 2 hours (the longest amount of time i was willing to wait)

### Screenshots

I have uploaded a youtube video depicting my situation.
(Yes i am going to watch the views to see if your looking into this or not lol /lh)
Video: https://youtu.be/THpPVAgF_Rc
Skip the first 17 seconds i didnt think to edit out the black in the beginning. 

### Resonite Version Number

Beta 2024.5.31.202

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Problem applies both in vr (quest 2) and desktop

### Log Files

[DESKTOP-T3R0JP4 - 2024.5.31.202 - 2024-06-02 16_18_06.log](https://github.com/user-attachments/files/15527107/DESKTOP-T3R0JP4.-.2024.5.31.202.-.2024-06-02.16_18_06.log)

### Additional Context

I asked for help in the new user assistance and they got me to do a clean install of my graphics drivers and the issue still persisted. The users that helped me say it might be simply an issue i have with my system but it would be a good idea to make a bug report just in case.

### Reporters

My discord is "thehuggblepanda." (with the period.) And the users that helped me (weren't able to reproduce the issue) are "euphieeuphoria", "karasutengu", and "reviver.noon"

**Comment by Thehuggblepanda** [comment]
I forgot to add that I've verified files multiple times with steam and each time there is one file thats not right and it says it failed to validate and will be re acquired.

**Comment by Thehuggblepanda** [comment]
Sorry thought i accidently closed it then actually closed it, This is my first time using github lol

**Comment by Thehuggblepanda** [comment]
Solved it, Deleted cache and it fixed it. The guesses of what happened was that the texture files were corrupted and was causing a gpu driver to crash. Might be a good idea to add a note for other users that might encounter similar problems.

**Comment by Thehuggblepanda** [comment]
Was told to keep this open until there was a proper solution so sorry for opening and closing so much.

**Comment by Frooxius** [comment]
Do you have the Player.log file for this? 

In the normal log I see that the GPU is suddenly removed, but there's no clue as to the cause of that happening.

**Comment by Thehuggblepanda** [comment]
I sadly do not. If I experience similar again I'll make sure to save it.

**Comment by Thehuggblepanda** [comment]
If you would like, I can see if I can try and replicate to the conditions that caused it to happen tonight

**Comment by Frooxius** [comment]
Yeah we need replication for this. Right now I don't have enough information to diagnose this.
